packages/@aws-cdk/aws-servicecatalogappregistry/README.md

```diff
@@ -124,3 +124,57 @@ const myStack = new Stack(app, 'MyStack');
 declare const application: appreg.Application;
 application.associateStack(myStack);
 ```
+
+## Sharing
+
+You can share your AppRegistry applications and attribute groups with AWS Organizations, Organizational Units (OUs), AWS accounts within an organization, as well as IAM roles and users. AppRegistry requires that AWS Organizations is enabled in an account before deploying a share of an application or attribute group.
+
+### Sharing an application
+
+```ts
+import * as iam from '@aws-cdk/aws-iam';
+declare const application: appreg.Application;
+declare const myRole: iam.IRole;
+declare const myUser: iam.IUser;
+application.shareResource({
+  accounts: ['123456789012'],
+  organizations: ['arn:aws:organizations::123456789012:organization/o-my-org-id'],
```

Comment:
Looks like an organization is not an object, but I'd like the opportunity to make it an object in the future. Can we call this `organizationArns` for now, so that if we ever have objects we can use the key `organizations` for that?

Reply:
I changed this to `organizationArns`.